test/src/org/omegat/gui/glossary/FindGlossaryThreadTest.java

```diff
@@ -43,7 +43,11 @@ public void testEntriesSort() {
         entries.add(new GlossaryEntry("cat", "mikeneko", "ccat", false, null));
         entries.add(new GlossaryEntry("zzz", "zzz", "czzz", true, null));
         entries.add(new GlossaryEntry("horse", "catty", "chorse", false, null));
+        entries.add(new GlossaryEntry("向上", "enhance", "", false, null));
+        entries.add(new GlossaryEntry("向", "direct", "", false, null));
+        entries.add(new GlossaryEntry("上", "up", "", false, null));
```

Comment:
OmegaT 6.0.x does not allow non ASCII UTF-8 characters in sources. It should be escaped.
(OmegaT 6.1 allows it)